### PR TITLE
Update Gradle and add custom background

### PR DESCRIPTION
**Pull request by lucas34**

### CODE REVIEW

**Comment by lucas34** [comment]
The most important is the second commit.

The background of the PDF was hardcoded to white. I changed it. Now default is White but you can set a 
custom background color. With : 

pdfView.fromFile(file).background(Color.BLACK).load();

The internal method is called setBackgroundPDF and not setBackground or setBackgroundColor because they already exist in the class View and I didn't want to override them.

File: android-pdfview/src/main/java/com/joanzapata/pdfview/PDFView.java [hunk 1]
```diff
@@ -1073,6 +1082,7 @@ public void load() {
             PDFView.this.maskPaint = new Paint();
             PDFView.this.maskPaint.setColor(maskColor);
             PDFView.this.maskPaint.setAlpha(maskAlpha);
+            PDFView.this.setBackgroundPDF(background);
```
Comment (fawaad) [inline android-pdfview/src/main/java/com/joanzapata/pdfview/PDFView.java]: Maybe something like `setPdfViewBackgroundColor(background)` instead? setBackgroundPDF isn't very intuitive.

Comment (lucas34) [inline android-pdfview/src/main/java/com/joanzapata/pdfview/PDFView.java]: I was not very inspired. 
This name is more intuitive. I pushed

**Comment by lucas34** [comment]
Don't want to merge ?

**Pull request closed without merging**